Command/InvalidateTagCommand.php

```diff
@@ -32,13 +38,14 @@ class InvalidateTagCommand extends BaseInvalidateCommand
      * If no cache manager is specified explicitly, fos_http_cache.cache_manager
      * is automatically loaded.
      *
-     * @param CacheManager|null $cacheManager The cache manager to talk to.
-     * @param string            $commandName  Name of this command, in case you want to reuse it.
+     * @param TagHandler|null $tagHandler  The tag handlerto talk to.
+     * @param string          $commandName Name of this command, in case you want to reuse it.
      */
-    public function __construct(CacheManager $cacheManager = null, $commandName = 'fos:httpcache:invalidate:tag')
+    public function __construct(TagHandler $tagHandler = null, $commandName = 'fos:httpcache:invalidate:tag')
```

Comment:
if people defined their own service for the InvalidateTagCommand or extended the command, this is a BC break. do we care about it or can we assume this an edge case? if we care, what are alternatives that make sense?